.github/workflows/ci.yml

```diff
@@ -76,3 +76,20 @@ jobs:
 
       - name: Test with all features
         run: cargo test --all-features --workspace
+
+      - name: Test with all features
+        run: cargo bench --all-features --workspace
```

Comment:
Isn't this a [duplicate of this](https://github.com/surrealdb/revision/pull/14/files#diff-b803fcb7f17ed9235f1e5cb1fcd2f5d3b2838429d4368ae4c57ce4436577f03fR95)?

Reply:
Totally